.jenkins/library/vars/common.groovy

```diff
@@ -0,0 +1,235 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/************************************
+* Shared Library for common functions
+************************************/
+
+String dockerBuildArgs(String... args) {
+    String argumentString = ""
+    for(arg in args) {
+        argumentString += " --build-arg ${arg}"
+    }
+    return argumentString
+}
+
+String dockerImage(String tag, String dockerfile = ".jenkins/Dockerfile", String buildArgs = "") {
+    return docker.build(tag, "${buildArgs} -f ${dockerfile} .")
+}
+
+def ContainerRun(String imageName, String compiler, String task, String runArgs="") {
+    exec_with_retry(10,300){
+        docker.withRegistry("https://oejenkinscidockerregistry.azurecr.io", "oejenkinscidockerregistry") {
```

Comment:
I understand that this is legacy code, but is there a reason why the URL and registry name are hardcoded for `ContainerRun` but not for `ContainerTasks` and `azureEnvironment`?

Reply:
I just did this anyways since it's a small change.